CONTRIBUTING.md

```diff
@@ -18,6 +18,7 @@ help and spread the knowledge.
 
 Contributions should follow these guidelines:
 
+* all development done in a personal fork;
```

Comment:
-> all development is done in a personal fork ? Sounds a bit more like a real sentence :-/